### PR TITLE
refs [TASK][28573] Create UI

### PR DESCRIPTION
**Pull request by peartalent**
## Related Tickets
- [#Task 28573: Create UI](https://edu-redmine.sun-asterisk.vn/issues/28573)

## WHAT
- Create UI detail movie
## Evidence (Screenshot or Video)
image

## Review Checklist

Category | View Point | Description | Expected Reviewer Answer | Self review | Reviewer2 (name)
--- | --- | --- | --- | --- | ---
General |Code should be self-explanatory. | Get a feel of story reading, while going through the code. Use appropriate name for variables, functions and classes. If you are taking more time to understand the code, then either code needs refactoring or at least comments have to be written to make it clear. | YES |<li>- [x] yes</li>|<li>- [ ] yes</li>  
General |Is this function or class too big? If yes, is the function or class having too many responsibilities? | | NO |<li>- [x] no</li>|<li>- [ ] no</li>  
General | Does the code follow DRY?: Do not Repeat Yourself | Is there any redundant or duplicate code? DRY (Do not Repeat Yourself) principle: The same code should not be repeated more than twice. Consider reusable services, functions and components. Consider generic functions and classes. | YES |<li>- [x] yes</li>|<li>- [ ] yes</li>  
General |Exception handling and cleanup (dispose) resources | | YES |<li>- [x] yes</li>|<li>- [ ] yes</li>  
General | Does the code build and execute correctly? | | YES |<li>- [x] yes</li>|<li>- [ ] yes</li>  
Conventions | Does the source code have end-line errors? | | NO |<li>- [x] no</li>|<li>- [ ] no</li>  
Conventions | No hard coding, use constants/configuration values | | YES |<li>- [x] yes</li>|<li>- [ ] yes</li>  
Conventions | Does the code follow Sun* coding style and coding conventions? | https://github.com/framgia/coding-standards/tree/master/eng/android | YES |<li>- [x] yes</li>|<li>- [ ] yes</li>  
Redmine | Does the ticket follow Sun* Redmine working process?  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md| YES |<li>- [x] yes</li>|<li>- [ ] yes</li>  
Documentation | Is there any incomplete code? If so, should it be removed or flagged with a suitable marker like ‘TODO’? |  | YES |<li>- [x] yes</li>|<li>- [ ] yes</li>  
## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(List gem, library third party add new)*

*(Other notes)*

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -9,6 +9,7 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/Theme.Design.NoActionBar">
+        <activity android:name=".ui.detailmovie.DetailMovie"></activity>
```
Comment (tuannm-1276) [inline app/src/main/AndroidManifest.xml]:
```suggestion
        <activity android:name=".ui.detailmovie.DetailMovie" />
```

File: app/src/main/java/com/sunasterisk/movieintroduction/ui/detailmovie/DetailMovie.kt [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package com.sunasterisk.movieintroduction.ui.detailmovie
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import com.sunasterisk.movieintroduction.R
+
+class DetailMovie : AppCompatActivity() {
```
Comment (tuannm-1276) [inline app/src/main/java/com/sunasterisk/movieintroduction/ui/detailmovie/DetailMovie.kt]: nếu em nhiều activity thì xem xét có một lớp BaseActivity

File: app/src/main/res/layout/activity_detail_movie.xml [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+    tools:context=".ui.detailmovie.DetailMovie">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <androidx.constraintlayout.widget.Guideline
+            android:id="@+id/guidelineNameMovie"
+            android:layout_width="0dp"
+            android:layout_height="match_parent"
```
Comment (tuannm-1276) [inline app/src/main/res/layout/activity_detail_movie.xml]: view con của Constraint ko có match_parent

File: app/src/main/res/layout/activity_detail_movie.xml [hunk 4]
```diff
@@ -0,0 +1,265 @@
...
+
+        <TextView
+            android:id="@+id/textOverviewContent"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="@dimen/dp_5"
+            android:layout_marginTop="@dimen/dp_5"
+            android:ellipsize="end"
+            android:maxLines="2"
+            android:padding="5dp"
```
Comment (tuannm-1276) [inline app/src/main/res/layout/activity_detail_movie.xml]: hardcode

File: app/src/main/res/values/colors.xml [hunk 5]
```diff
@@ -4,6 +4,8 @@
     <color name="colorPrimaryDark">#3700B3</color>
     <color name="colorAccent">#03DAC5</color>
     <color name="colorSaffron">#F4C124</color>
+    <color name="colorSaffron113">#71F4C124</color>
```
Comment (tuannm-1276) [inline app/src/main/res/values/colors.xml]: các tên trong này em đặt chưa đúng convention thì phải. xem lại

File: app/src/main/java/com/sunasterisk/movieintroduction/ui/detailmovie/DetailMovie.kt (hunk 2 above)
Comment (peartalent) [reply app/src/main/java/com/sunasterisk/movieintroduction/ui/detailmovie/DetailMovie.kt]: > nếu em nhiều activity thì xem xét có một lớp BaseActivity

Tại em chưa muốn làm bên code ạ, hôm nay em định hoàn thành các file xml

File: app/src/main/res/layout/activity_detail_movie.xml [hunk 6]
```diff
@@ -0,0 +1,263 @@
...
+            app:menu="@menu/menu_toolbar_detail_movie"
+            app:titleTextColor="@color/colorCapeCod" />
+
+        <ImageView
+            android:id="@+id/imagePoster"
+            android:layout_width="match_parent"
+            android:layout_height="0dp"
+            android:background="@drawable/bg_toolbar"
+            android:scaleType="fitCenter"
+            android:src="@drawable/backgroud"
```
Comment (tuannm-1276) [inline app/src/main/res/layout/activity_detail_movie.xml]:
```suggestion
            android:src="@drawable/background"
```

File: app/src/main/res/values/colors.xml [hunk 7]
```diff
@@ -4,6 +4,8 @@
     <color name="colorPrimaryDark">#3700B3</color>
     <color name="colorAccent">#03DAC5</color>
     <color name="colorSaffron">#F4C124</color>
+    <color name="colorMoonGlow">#FBFED2</color>
```
Comment (tuannm-1276) [inline app/src/main/res/values/colors.xml]:
```suggestion
    <color name="color_moon_glow">#FBFED2</color>
```
em đặt lại tên tương tự với các color khác

**Pull request merged**